lib/components/artist/artworks/grid.js

```diff
@@ -12,11 +12,12 @@
 import Relay from 'react-relay'
 import React from 'react'
 import { Dimensions, View, ScrollView, StyleSheet } from 'react-native'
+import { NetworkError } from '../../../system/errors'
 
 import Artwork from './artwork'
 import Spinner from '../../spinner'
 import { metaphysicsURL } from '../../../relay/config'
-
+import type { LayoutEvent } from '../../../system/events'
 
```

Comment:
Two new types that deal with how we typically handle events and errors, wasn't too sure where to host these, so I figured system would make sense as they're used in a bunch of different places